modules/ppcp-button/resources/js/modules/ErrorHandler.js

```diff
@@ -73,11 +73,13 @@ class ErrorHandler {
 
     clear()
     {
-        if (! this.wrapper.classList.contains('woocommerce-error')) {
+        if (this.messagesList === null) {
             return;
         }
-        this.wrapper.classList.remove('woocommerce-error');
-        this.wrapper.innerText = '';
+
+        while( this.messagesList.firstChild ){
+            this.messagesList.removeChild( this.messagesList.firstChild );
+        }
```

Comment:
maybe cleaner to just `.innerHTML = ''`?

https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML#replacing_the_contents_of_an_element

Reply:
Done :)